src/Microsoft.Azure.SignalR.Emulator/Common/Constants.cs

```diff
@@ -37,5 +37,18 @@ public static class Headers
             public const string AsrsClientCertThumbprint = AsrsHeaderPrefix + "Client-Cert-Thumbprint";
             public const string AsrsConnectionGroups = AsrsHeaderPrefix + "Connection-Group";
         }
+
+        public static class ErrorCodes
+        {
+            public const string WarningConnectionNotExisted = "Warning.Connection.NotExisted";
```

Comment:
What about add some `Enum` for `level`, `scope`, `errorKind` and use a method to build a ErrorCode like `level.scope.errorKind` instead of fully using a constant?